files/k8s-resource/calico-policy-only.yaml

```diff
@@ -351,41 +351,6 @@ spec:
       # deletion": https://kubernetes.io/docs/concepts/workloads/pods/pod/#termination-of-pods.
       terminationGracePeriodSeconds: 0
       priorityClassName: system-node-critical
-      initContainers:
-        # This container installs the CNI binaries
-        # and CNI network config file on each node.
-        - name: install-cni
-          image: {{ .Images.CalicoCNI }}
```

Comment:
Can also the image configuration be removed?

Reply:
image is still necessary for KVM ( and probabbly azure)

Reply:
I have disabled only for AWS CNI now